src/pages/docs/data-folder.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Jan Data Folder
+description: Discover the Structure of Jan Data.
+sidebar_position: 2
+keywords:
+  [
+    Jan,
+    Customizable Intelligence, LLM,
+    local AI,
+    privacy focus,
+    free and open source,
+    private and offline,
+    conversational AI,
+    no-subscription fee,
+    large language models,
+    quickstart,
+    getting started,
+    using AI model,
+  ]
+---
+
+import { Tabs } from 'nextra/components'
+import { Callout, Steps } from 'nextra/components'
+
+# Jan Data Folder
+
+After installing Jan, a structured data folder is created on the local filesystem. This setup promotes an organized approach to data management. Below is a guide to understanding how files are organized within Jan data folder.
+
+## Folder Structure
+Jan app data folder should have the following folder structure:
+```yaml
+jan/                                
+  assistants/                           
+  engines/                              
+  logs/                                 
+  models/                               
+  settings/                             
+  threads/                              
+  extensions/                           
+```
+### `jan/` (The Root Directory)
+
+This is the primary directory where all files related to Jan are stored. It typically resides in the user's home directory.
+
+### `assistants/`
```

Comment:
Can't we combine the "Folder Structure" and "File Structure" sections?

Especially since these sections are so short and we repeat this exact information again in the "Detailed File Layout" section